<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>主窗口</string>
       </attribute>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>12</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>上限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>下限</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>80</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>周期</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>130</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No.0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No.1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No.2</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>69</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>130</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>270</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_5">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>11</y>
          <width>69</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string>com4</string>
        </property>
        <item>
         <property name="text">
          <string>com1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>com8</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>记录窗口</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>511</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
